### DISCUSSION TITLE
Programs take a long time to show.

### DISCUSSION THREAD

**Discussion by Eugen-str** [i3/i3#5592] [Q&A]
Programs like Firefox or Nautilus take way too long to show (up to 40 seconds). I know it's not hardware or anything because on GNOME they show immediately after i start them. Also things like Emacs or Alacritty and even Chromium turn on almost immediately. 

for example: 
$ time nautilus

real	0m38,703s
user	0m0,478s
sys	0m0,030s

I stopped nautilus immediately after it showed so it took roughly 37 seconds to show up.

Also, not sure if this means anything but when i have a Firefox tab already open, I can open a new tab very quickly. It's just when I turn off every Firefox window it takes a long time to open a new one.

(I'm new to linux so sorry if I'm missing something obvious...)

**Comment by pietermartin** [comment]
I found this link but have not tried it yet.
https://streamofletters.xyz/fix-for-firefoxs-slow-start-on-arch-linux
